omero/sysadmins/server-upgrade.txt

```diff
@@ -280,6 +280,31 @@ intermediate major release.
    connected to the database using **db_user** before running the script. See
    :forum:`this forum thread <viewtopic.php?f=5&t=7778>` for more information.
 
+Delete certain annotations (optional)
+"""""""""""""""""""""""""""""""""""""
+
+For various reasons production databases may accumulate non-sharable
+annotations that are orphaned. These are :doc:`structured annotations
+</developers/Model/StructuredAnnotations>` that are 'basic' (`Boolean`,
+`TimeStamp`, `Term`), 'numeric' (`Double`, `Long`), or `Comment`, and
+that are *not* annotating an object. An illustrative example is that
```

Comment:
``...annotating an object.`` Reformulate : ``Non-sharable orphaned annotations are :doc:`structured annotations
+</developers/Model/StructuredAnnotations>` that are 'basic' (`Boolean`,`TimeStamp`, `Term`), 'numeric' (`Double`, `Long`), or `Comment`, which are *not* annotating an object.